LaTeX/TeX.sublime-syntax

```diff
@@ -148,21 +171,67 @@ contexts:
         - include: main
 
   macros:
-    - match: ((\\)def)\s*((\\)[A-Za-z@]+)\s*[^\{]*?\s*(\{)
+    # Note \edef and \xdef have slightly different syntax, in that they do not
+    # allow for arguments. However, every valid \edef could be a valid \def,
+    # so syntax highlighting will work properly with correct code.
+    # we need to make sure to match only full commands, so we look-ahead for
+    # any following non-letter
+    - match: (\\)[gex]?def(?={{nonletter}})
```

Comment:
I'd suggest a variable like `{{break}}` which is defined as `(?!{{letter}})` to also use the strategy of negative lookahead as in LaTeX.

Reply:
this also gives us the option of ensuring that the other commands/keywords actually terminate correctly, instead of \b which misses that an `@` can continue a command name.